integration/delete_compat_test.go

```diff
@@ -41,18 +41,82 @@ func TestDeleteCompat(t *testing.T) {
 			filters:    []bson.D{},
 			resultType: emptyResult,
 		},
+		"OrderedTrue": {
+			filters: []bson.D{
+				{{"_id", "string"}},
```

Comment:
The whole point of compatibility tests is to not have IDs hard-coded in the test. We run commands for all data sets and collections. We could depend on data staying the same, but we can't depend on IDs